integration/__tests__/app.tests.ts

```diff
@@ -97,21 +113,21 @@ describe("Lens integration tests", () => {
   const ready = minikubeReady()
 
   const addMinikubeCluster = async (app: Application) => {
-    await app.client.click("div.add-cluster")
+    await click("div.add-cluster")
     await app.client.waitUntilTextExists("div", "Select kubeconfig file")
-    await app.client.click("div.Select__control") // show the context drop-down list
+    await click("div.Select__control") // show the context drop-down list
     await app.client.waitUntilTextExists("div", "minikube")
-    if (!await app.client.$("button.primary").isEnabled()) {
-      await app.client.click("div.minikube") // select minikube context
+    if (!(await app.client.$("button.primary")).isEnabled()) {
+      await click("div.minikube") // select minikube context
     } // else the only context, which must be 'minikube', is automatically selected
-    await app.client.click("div.Select__control") // hide the context drop-down list (it might be obscuring the Add cluster(s) button)
-    await app.client.click("button.primary") // add minikube cluster
+    await click("div.Select__control") // hide the context drop-down list (it might be obscuring the Add cluster(s) button)
+    await click("button.primary") // add minikube cluster
   }
 
   const waitForMinikubeDashboard = async (app: Application) => {
     await app.client.waitUntilTextExists("pre.kube-auth-out", "Authentication proxy started")
-    await app.client.waitForExist(`iframe[name="minikube"]`)
-    await app.client.frame("minikube")
+    await (await app.client.$(`iframe[name="minikube"]`)).waitForExist()
+    await app.client.switchToFrame("minikube")
```

Comment:
Build job shows
```
    Malformed type for "id" parameter of command switchToFrame
    Expected: (number|object|null)
    Actual: string
```
When I hover in VS Code it thinks it should be `id: string | number | object` so deps may still be off?

According to docs it wants the element's id or you can pass the object (gotten from $). I tried this but it just times out:
```suggestion
    const iframeElement = await app.client.$(`iframe[name="minikube"]`)
    await iframeElement.waitForExist()
    await app.client.switchToFrame(iframeElement)
```